content/en/monitors/downtimes/_index.md

```diff
@@ -114,7 +114,7 @@ Monitors trigger events when they change between possible states: `ALERT`, `WARN
 
 By default, if a monitor is in an alert-worthy state (`ALERT`, `WARNING`, or `NO DATA`) when a downtime expires, the monitor triggers a new notification. This applies to monitors that change state during downtime (such as from `OK` to `ALERT`, `WARNING`, or `NO DATA`), and to monitors that already have an alert-worthy state when downtime begins. If a downtime is manually canceled, notifications are not sent, even if the monitor has entered an alert-worthy state.
 
-To override the default behavior, specify which notifications should be sent at the end of downtimes with the options in the "Notify Your Team" section.
+To override the default behavior, specify which notifications should be sent at the end of downtimes with the options in the "Notify Your Team" section. For downtimes created via API, the the default behaviour is to exclude is_cancelled option.
```

Comment:
```suggestion
To override the default behavior, specify which notifications should be sent at the end of downtimes with the options in the "Notify Your Team" section. For downtimes created with the API, the the default behavior is to exclude `is_cancelled` option.
```

Reply:
Good catch! I updated the comment.
@sou999 you should be good to go ahead and commit the suggested changes.